### DISCUSSION TITLE
Etsy Marketing Endpoints

### DISCUSSION THREAD

**Discussion by supermandee** [etsy/open-api#1229] [Q&A]
Hello, i've been trying to use the API to calculate my profits for each product. and with that i need to get the etsy Ads spend for each product; the API doesn't seem to have any endpoints for Etsy Marketing, however, there's no way i can get the clicks, views, orders etc from Etsy Marketing; 
Why is that?

**Comment by Gareth064** [comment]
Etsy simply haven't built a public API for that information.

It's been spoken about in here a few times, but nothing from Etsy to say they are going to do anything about it.
